resources/scss/componets/15_progress.scss

```diff
@@ -0,0 +1,19 @@
+.progress{
```

Comment:
this is a direct **bootstrap class**
do not overlap same class 
name it separately with meaning like 

.progress-mini
.progress-gradient
.progress-theme

like these

Reply:
Fixed